languages.md

```diff
@@ -1,34 +1,56 @@
+
+By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling. To override the measurement system used in spoken instructions, set the `RouteOptions.measurementSystem` property when calculating the route with which to start navigation.
+
+The upcoming road or ramp destination is named according to the local or national language. In some regions, the name may be given in multiple languages.
+
+## Supported languages
+
+| Language   | User interface | [Spoken instructions][osrmti] | [Amazon Polly][polly] | [`AVSpeechSynthesizer`][iossynth]<br>(iOS 11)
+|------------|----------------|-------------------------------|-----------------------|----------------------------------
+| Catalan    | ✅              | ❌                             | ❌                     | ❌
+| Chinese    | ✅ Simplified   | ✅                             | ❌                     | ✅
+| Danish     | ✅              | ❌                             | ❌                     | ✅
+| Dutch      | ✅              | ✅                             | ✅                     | ✅
+| English    | ✅              | ✅                             | ✅                     | ✅
+| French     | ✅              | ✅                             | ✅                     | ✅
+| German     | ✅              | ✅                             | ✅                     | ✅
+| Hebrew     | ✅              | ❌                             | ❌                     | ✅
+| Hungarian  | ✅              | ❌                             | ❌                     | ✅
+| Italian    | ✅              | ✅                             | ✅                     | ✅
+| Portuguese | ✅              | ✅                             | ✅                     | ✅
+| Polish     | ❌              | ✅                             | ✅                     | ✅
+| Romanian   | ❌              | ✅                             | ✅                     | ✅
+| Russian    | ✅              | ✅                             | ✅                     | ✅
+| Spanish    | ✅              | ✅                             | ✅                     | ✅
+| Swedish    | ✅              | ✅                             | ✅                     | ✅
+| Turkish    | ❌              | ✅                             | ✅                     | ✅
+| Vietnamese | ✅              | ✅                             | ❌                     | ❌
+
+## Contributing
+
+See the [contributing guide](./CONTRIBUTING.md#adding-or-updating-a-localization) for instructions on adding a new localization or improving an existing localization.
+
+[osrmti]: https://github.com/Project-OSRM/osrm-text-instructions/
```

Comment:
Are these lines supposed to be hidden?

Reply:
Yes, this is an alternative way to create a hyperlink in Markdown. Any occurrence of `[…][osrmti]` above is linked to [this page](https://github.com/Project-OSRM/osrm-text-instructions/). Meanwhile, this line is hidden from the rendered page. I went with this syntax because linking text within a table header causes the table to become fairly unreadable.